package.json

```
@@ -106,7 +106,11 @@
       "^.+\\.(t|j)sx?$": [
         "@swc/jest"
       ]
-    }
+    },
+    "testPathIgnorePatterns": [
+      "/node_modules/",
+      "<rootDir>/test/integration/"
```

Only skip source folder
```suggestion
      "<rootDir>/test/integration/.*/*src"
```